The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start

waiting for server to start....2025-02-14 13:10:54.042 UTC [52] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:10:54.049 UTC [53] LOG:  database system was shut down at 2025-02-14 13:10:53 UTC
2025-02-14 13:10:54.052 UTC [52] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 13:10:54.394 UTC [52] LOG:  received fast shutdown request
2025-02-14 13:10:54.395 UTC [52] LOG:  aborting any active transactions
2025-02-14 13:10:54.396 UTC [52] LOG:  background worker "logical replication launcher" (PID 59) exited with exit code 1
2025-02-14 13:10:54.396 UTC [54] LOG:  shutting down
2025-02-14 13:10:54.404 UTC [52] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 13:10:54.502 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 13:10:54.502 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 13:10:54.508 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:10:54.517 UTC [70] LOG:  database system was shut down at 2025-02-14 13:10:54 UTC
2025-02-14 13:10:54.527 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 13:11:01.942 UTC [85] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 13:11:01.942 UTC [85] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 13:11:02.234 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:02.234 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_c0226fdc-a696-4069-b1e5-d7a849159a25" WITH (FORCE)
2025-02-14 13:11:02.703 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:02.703 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_1a928b72-c6be-406d-8ba7-f667e2f08aff" WITH (FORCE)
2025-02-14 13:11:03.116 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:03.116 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_7fb9d262-897e-401d-a776-64a8482582c3" WITH (FORCE)
2025-02-14 13:11:03.608 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:11:03.608 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_bec2062e-a1c4-411a-9c81-759276077efe" WITH (FORCE)
